folium/_server.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+
+"""
+Simple HTTP server so we are able to show Maps in browser. Only works with python 3!.
+
+"""
+
+import sys
+if sys.version_info[0] < 3:
+    raise NotImplementedError('Only works with python 3!')
```

Comment:
Maybe a more verbose message like:

`This feature is only available on Python 3.`

Reply:
I actually realised i might have misunderstood something. Because there is an http server in python 2, so I'll check it again tomorrow.